packages/lesswrong/lib/collections/users/helpers.ts

```diff
@@ -164,6 +164,10 @@ export const userIsAllowedToComment = (user: UsersCurrent|DbUser|null, post: Pos
     return false
   }
 
+  if (user.deleted) {
+    return false
+  }
```

Comment:
Same comment here, although here it at least looks like a this function is already being used for this. But you could you still imagine that this is an enforcement against all user new comments and being enforced elsewhere.